packages/react-scripts/package.json

```diff
@@ -94,6 +94,7 @@
     "fsevents": "^2.1.3"
   },
   "peerDependencies": {
+    "react": "^17.0.0",
```

Comment:
Which versions do you actually want here?

Reply:
I think we could give it a range that starts our out minimum supported version (which is 16.x I guess).